source/list_utils.c

```diff
@@ -35,13 +40,17 @@ int aws_cryptosdk_transfer_list(struct aws_array_list *dest, struct aws_array_li
 typedef int (*clone_item_fn)(struct aws_allocator *, void *, const void *);
 typedef void (*clean_up_item_fn)(void *);
 
-static int list_copy_all(
+/*static*/ int list_copy_all(
```

Comment:
Why? If we are commenting out, why don't just remove it?

Reply:
changed the makefiles so we can use the static version